.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+# Describe what this PR does #
+
+Provide some context for the reviewer
+
+## Is there anything that requires special attention ##
+
+Do you have any questions?
+
+Is the change backward compatible?
+
+Are there concerns around backward compatibility?
+
+Provide any external context for the change, if any.
+
+For example:
+
+    * Kubernetes links that explain why the change is required
+    * CSI spec related changes/catch-up that necessitates this patch
+    * golang related practices that necessitates this change"
+
+## Related issues ##
+
+Mention any github issues relevant to this PR
```

Comment:
add only one more line for `Fixes: #issue_number`  for auto-closing the issue once PR is merged

Reply:
done.